src/Microsoft.TemplateEngine.Edge/Template/TemplateCreator.cs

```diff
@@ -106,7 +106,6 @@ public IReadOnlyCollection<IFilteredTemplateInfo> List(bool exactMatchesOnly, pa
                 IComponentManager componentManager = _environmentSettings.SettingsLoader.Components;
                 creationResult = await template.Generator.CreateAsync(_environmentSettings, template, templateParams, componentManager, targetDir).ConfigureAwait(false);
                 sw.Stop();
-                _environmentSettings.Host.OnTimingCompleted("Content generation time", sw.Elapsed);
```

Comment:
Works for me. It could also come into play in a verbose debugging mode, as we've touched on in the past.